codespell_lib/data/dictionary.txt

```diff
@@ -5888,6 +5893,7 @@ extry->entry
 eyar->year, eyas,
 eyars->years, eyas,
 eyasr->years, eyas,
+facedrwaing->facedrawing, face drawing,
```

Comment:
I'm aware this is non-standard. Nevertheless, it's out there: https://github.com/search?q=facedrwaing&type=Code